aws_lambda_powertools/utilities/parameters/ssm.py

```diff
@@ -811,6 +941,81 @@ def get_parameters(
     )
 
 
+def set_parameter(
+    name: str,
+    value: str,
+    *,  # force keyword arguments
+    overwrite: bool = False,
+    description: str = "",
+    parameter_type: SSM_PARAMETER_TYPES = "String",
+    tier: SSM_PARAMETER_TIER = "Standard",
+    kms_key_id: str | None = None,
+    **sdk_options,
+) -> PutParameterResponse:
+    """
+    Sets a parameter in AWS Systems Manager Parameter Store.
+
+    Parameters
+    ----------
+    name: str
+        The fully qualified name includes the complete hierarchy of the parameter name and name.
+    value: str
+        The parameter value
+    parameter_type: str, optional
+        Type of the parameter.  Allowed values are String, StringList, and SecureString
+    overwrite: bool, optional
+        If the parameter value should be overwritten, False by default
+    tier: str, optional
+        The parameter tier to use.  Allowed values are Standard, Advanced, and Intelligent-Tiering
+    description: str, optional
+        The description of the parameter
+    kms_key_id: str, optional
+        The KMS key id to use to encrypt the parameter
+    sdk_options: dict, optional
+        Dictionary of options that will be passed to the Parameter Store get_parameter API call
```

Comment:
Same order as declared on the signature:

```suggestion
        name: str
            The fully qualified name includes the complete hierarchy of the parameter name and name.
        value: str
            The parameter value
        overwrite: bool, optional
            If the parameter value should be overwritten, False by default
        description: str, optional
            The description of the parameter
        parameter_type: str, optional
            Type of the parameter.  Allowed values are String, StringList, and SecureString
        tier: str, optional
            The parameter tier to use.  Allowed values are Standard, Advanced, and Intelligent-Tiering
        kms_key_id: str, optional
            The KMS key id to use to encrypt the parameter
        sdk_options: dict, optional
            Dictionary of options that will be passed to the Parameter Store get_parameter API call
```